wandb/sdk/artifacts/staging.py

```diff
@@ -0,0 +1,18 @@
+import os
```

Comment:
Nit: let's make it a habit to add a docstring at the top of every file. It helps even if it's trivial.